build.sh

```diff
@@ -65,7 +65,7 @@ function build_image_z690 {
 	docker run --rm -t -u $UID -v $PWD:/home/coreboot/coreboot \
 		-v $HOME/.ssh:/home/coreboot/.ssh \
 		-w /home/coreboot/coreboot coreboot/coreboot-sdk:2021-09-23_b0d87f753c \
-		/bin/bash -c "make olddefconfig && make -j$(nproc)"
```

Comment:
In what way the build breaks? @SergiiDmytruk 

I would like to keep the multiple jobs, otherwise the CI may make even longer to build (not even mentioning users)

Reply:
There was some compilation error, restarting the build helped as it doesn't happen every time.  `make` also hanged after doing some changes and recompiling.

If it works more often than not, I'll just keep the change locally and will take a closer look if the failure repeats, I just thought `-j` was added by accident.

Reply:
I have never experienced such issues TBH. Whenever I repeat the make after compilation issues it always builds with the fresh local changes.